lib/roo_on_rails/railties/routemaster.rb

```diff
@@ -0,0 +1,36 @@
+require 'roo_on_rails/config'
+
+module RooOnRails
+  module Railties
+    class Routemaster < Rails::Railtie
+      initializer 'roo_on_rails.routemaster' do
+        next unless Config.routemaster_enabled?
+
+        $stderr.puts 'initializer roo_on_rails.routemaster'
+
+        abort 'Aborting: ROOBUS_URL and ROOBUS_UUID are required' if roobus_credentials_blank?
+
+        require 'routemaster/client'
+
+        ::Routemaster::Client.configure do |config|
+          config.url = roobus_url
+          config.uuid = roobus_uuid
+        end
+      end
+
+      private
+
+      def roobus_credentials_blank?
+        roobus_url.blank? && roobus_uuid.blank?
+      end
+
+      def roobus_url
+        ENV.fetch('ROOBUS_URL')
```

Comment:
I just assumed that these were prefixed with `ROOBUS_` for a good reason and didn't even question it, but it would definitely make sense to be more consistent and use `ROUTEMASTER_`. Happy to make this change if people agree.